pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -103,7 +104,18 @@ func TestDefaultConfigMap(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if diff := deep.Equal(DefaultConfigMap(tt.args.operatorConfig, tt.args.consoleConfig, tt.args.rt), tt.want); diff != nil {
+			cm, _, _, _ := DefaultConfigMap(tt.args.operatorConfig, tt.args.consoleConfig, tt.args.rt)
+
```

Comment:
@zherman0 @spadgett 
This is a tricky one.  The embedded yaml string really can't be tested well due to unstable ordering (from JSON conversion).  Fortunately, we have the other `TestNewYamlConfig()` func below.

Perhaps this is the way to get these changes unstuck.

Reply:
The problem here is that its an obvious hack and it mutates the output.
Also, if the func is changed & for some reason the `cm.Data` doesn't return a string, we miss that.  So we could miss a legitimate error.